### ISSUE TITLE
Test failures in stable-4.9 branch of GAP

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/loops#3]
It will be nice to release the new version of LOOPS since LOOPS 3.4.0 does not comply with automated test failure detection, so nobody is alerted when changes in GAP break the test. I've just discovered that LOOPS tests fail when all packages are loaded with the error saying
```
if not x ^ G!.FRData.pi in Image( G!.FRData.pi ) then
    return false;
elif ForAny( G!.FRData.sphere, function ( s )
          return x in s;
      end ) then
    return true;
fi; at /home/gap/inst/gap-stable-4.9/pkg/fr-2.4.5/gap/group.gi:166
``` 
Please see https://travis-ci.org/gap-system/gap-docker-pkg-tests-stable-4.9-staging/jobs/417080418

**Comment by nagygp** [comment]
@alex-konovalov Can we close this issue after solving #4 ?

**Comment by olexandr-konovalov** [comment]
Sure. Problem resolved, and we do not test stable-4.9 branch any more.
